index.html

```diff
@@ -1265,6 +1268,15 @@ <h3>Video format support (last update July 2020)</h3>
             }
           },
 
+	  copySubtitle: function () {
+	    var helpertextarea = document.createElement("textarea");
+	    document.body.appendChild(helpertextarea);
+	    helpertextarea.value = document.getElementsByClassName('current-caption')[0].textContent;
+	    helpertextarea.select();
+	    document.execCommand("copy");
+	    document.body.removeChild(helpertextarea);
```

Comment:
It'd be nice to have some visual feedback so you know you pushed the right key.
```suggestion
	    document.body.removeChild(helpertextarea);
	    self.notify("Copied current subtitle to clipboard");
```